specification/agfood/resource-manager/Microsoft.AgFoodPlatform/preview/2020-05-12-preview/agfood.json

```diff
@@ -0,0 +1,953 @@
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/CheckNameAvailabilityResponse"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Locations_CheckNameAvailability_AlreadyExists": {
+            "$ref": "./examples/Locations_CheckNameAvailability_AlreadyExists.json"
+          },
+          "Locations_CheckNameAvailability_Available": {
+            "$ref": "./examples/Locations_CheckNameAvailability_Available.json"
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.AgFoodPlatform/operations": {
+      "get": {
+        "tags": [
+          "Operations"
+        ],
+        "description": "Lists the available operations of Microsoft.AgFoodPlatform resource provider.",
+        "operationId": "Operations_List",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/OperationListResult"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "Operations_List": {
+            "$ref": "./examples/Operations_List.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ExtensionProperties": {
+      "description": "Extension resource properties.",
+      "type": "object",
+      "properties": {
+        "extensionId": {
+          "description": "Extension Id.",
+          "pattern": "^[A-za-z]{3,50}[.][A-za-z]{3,100}$",
+          "type": "string",
+          "readOnly": true
+        },
+        "extensionCategory": {
+          "description": "Extension category. e.g. weather/sensor/satellite.",
+          "type": "string",
+          "readOnly": true
+        },
+        "installedExtensionVersion": {
+          "description": "Installed extension version.",
+          "pattern": "^([1-9]|10).\\d$",
+          "type": "string",
+          "readOnly": true
+        },
+        "extensionAuthLink": {
+          "description": "Extension auth link.",
+          "type": "string",
+          "readOnly": true
+        },
+        "extensionApiDocsLink": {
+          "description": "Extension api docs link.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "ErrorResponse": {
```

Comment:
Better to refer to v2/types.json#definitions/ErrorResponse

Reply:
updated